### PR TITLE
Update zh_CN file

### PR DESCRIPTION
**Pull request by TartaricAcid**
Before the pull request be merged, I want to ask @SihenZhang to make a check on it.

@SihenZhang Please make a approve if it don't have mistake.

### CODE REVIEW

File: src/main/resources/assets/actuallyadditions/lang/zh_CN.lang [hunk 1]
```diff
@@ -550,15 +503,22 @@ tooltip.actuallyadditions.laser.stored.desc=连接开始！
...
+tooltip.actuallyadditions.itemBooklet.sub.3=提供的特性。
+tooltip.actuallyadditions.itemBooklet.sub.4=手持右键即可打开
 tooltip.actuallyadditions.playerProbe.disconnect.1=那位被探测的玩家在你试图把他和玩家接口绑定之前中断了连接！数据将会被清除！
 tooltip.actuallyadditions.playerProbe.disconnect.2=那位被探测的玩家在你试图把他和玩家接口绑定之前中断了连接！数据将会被清除！
 tooltip.actuallyadditions.playerProbe.probing=探测中
 tooltip.actuallyadditions.playerProbe.notice=注意！有人正在试图探测你并且想把你和玩家接口绑定，但是失败了！
 tooltip.actuallyadditions.battery.discharge=会给物品栏中的其他物品充电
 tooltip.actuallyadditions.battery.noDischarge=不会给物品栏中的其他物品充电
 tooltip.actuallyadditions.battery.changeMode=Shift右键转换两种模式
+tooltip.actuallyadditions.previouslyDoubleFurnace=以前的“量仓熔炉”
```
Comment (SihenZhang) [inline src/main/resources/assets/actuallyadditions/lang/zh_CN.lang]: 以前的“两仓熔炉”

File: src/main/resources/assets/actuallyadditions/lang/zh_CN.lang [hunk 2]
```diff
@@ -758,12 +717,14 @@ booklet.actuallyadditions.chapter.quartz.text.2=<item>焦黑石英<r>在<imp>实
 
 booklet.actuallyadditions.chapter.cloud.name=微笑云朵
 booklet.actuallyadditions.chapter.cloud.text.1=<item>微笑云朵<r>是一种有魔力的浮动的云朵，通过它体内的固态经验获得生命。它在它呆的位置上上下下、兴高采烈地浮动着。当你<imp>右键点击<r>它时，你可以<imp>给它命名<r>。<n>在给它某些<imp>特殊的名字<r>时，比如说<imp>"Ellpeck"<r>或<imp>"AcidBlues"<r>,它就会拿出它携带的某些特殊的东西！
+booklet.actuallyadditions.chapter.cloud.text.2=<n><n><n><i>Huge potato
```
Comment (SihenZhang) [inline src/main/resources/assets/actuallyadditions/lang/zh_CN.lang]: 这里怀疑玩的是Botania的梗，Botania有个卖萌用的小土豆，本质跟微笑云朵相似，如果Botania是小土豆，那微笑云朵就称得上是“巨型土豆”了

File: src/main/resources/assets/actuallyadditions/lang/zh_CN.lang [hunk 3]
```diff
@@ -906,10 +869,13 @@ booklet.actuallyadditions.chapter.hairBalls.name=毛球
 booklet.actuallyadditions.chapter.hairBalls.text.1=<item>毛球<r>会从<imp>活<r>猫身上掉落。<n>浏览下一页以获取更多信息。
 booklet.actuallyadditions.chapter.hairBalls.text.2=<item>毛球<r>是一件会从被玩家<imp>驯服<r>的<imp>猫<r>身上掉落的物品。<n>猫是一种很古怪的的生物，因为它们<imp>会让任何东西粘在自己的毛发上<r>。<item>毛球<r>或许长相令人不堪入目，但当你通过<imp>右击<r>移除上面那些杂乱的猫毛后，说不定会找到一些<imp>有价值的物品<r>。<n>或者只会收获到一些线。<n><n><n><i>FUURRRRRR！！
 
+booklet.actuallyadditions.chapter.laserIntro.name=激光中继器的介绍
+booklet.actuallyadditions.chapter.laserIntro.text.1=<item>激光中继器<r>存在<imp>很多变种<r>，但是都拥有相似的功能。使用<item>中继器扳手<r>能够将<n>一个<item>激光中继器<r>和其他中继器相连。只需要<imp>右击<r>第一个中继器后在右击另一个即可连接。连接<imp>没有上限<r>，你可以称呼它为<item>网络<r>或者<item>系统<r>。这样的网络系统没有预定的终点或者起点。
```
Comment (SihenZhang) [inline src/main/resources/assets/actuallyadditions/lang/zh_CN.lang]: 只需要<imp>右击<r>第一个中继器后再右击另一个即可连接。

File: src/main/resources/assets/actuallyadditions/lang/zh_CN.lang [hunk 4]
```diff
@@ -1022,3 +991,38 @@ booklet.actuallyadditions.chapter.empowerer.text.1=<item>充能台<r>是一个
 booklet.actuallyadditions.chapter.empowerer.text.2=<i><item>物品展示台<r><i>的布局
 booklet.actuallyadditions.chapter.empowerer.text.3=<n><n><n><i>我想所有MOD都得来个注魔祭坛
 booklet.actuallyadditions.chapter.empowerer.text.4=<item>物品展示台<r>也可以用来做很多其他的事情！翻到<imp>耗能方块<r>查询更多信息！ 
+
+booklet.actuallyadditions.chapter.fluidLaser.name=流体激光中继器
+booklet.actuallyadditions.chapter.fluidLaser.text.1=<item>流体激光中继器<r>工作方式颇有点像普通的<item>激光中继器<r>，但是和<item>流体激光中继器<r>和<item>能量激光中继器<r>又有些不同,，它只能把<imp>流体<r>从一个储罐传输到另一个储罐。<n><n><item>流体激光中继器<r>不能连接幻灵接口。
```
Comment (SihenZhang) [inline src/main/resources/assets/actuallyadditions/lang/zh_CN.lang]: <item>流体激光中继器<r>工作方式颇有点像普通的<item>激光中继器<r>，但是和<item>流体激光中继器<r>和<item>能量激光中继器<r>又有些不同

这里很明显有问题，但是我没看过原文不太清楚

Comment (SihenZhang) [inline src/main/resources/assets/actuallyadditions/lang/zh_CN.lang]: 此外，两个逗号务必删掉一个

File: src/main/resources/assets/actuallyadditions/lang/zh_CN.lang [hunk 5]
```diff
@@ -1022,3 +991,38 @@ booklet.actuallyadditions.chapter.empowerer.text.1=<item>充能台<r>是一个
...
+booklet.actuallyadditions.chapter.patreon.text.1=你是否<imp>非常喜欢<r> <item>Actually Additions<r>这个模组？如果是这样的话，你可以通过如下方式<imp>支持<r>Ellpeck，也就是这个模组的作者哦！<n>只需要轻轻点击下面这个按钮，就可以打开<item>Patreon<r>页面！<n>正如下一页你所见的那样,作为奖励，你的头上会出现一些<imp>悬浮的小玩意<r>。<n><n>非常感谢<3
+booklet.actuallyadditions.chapter.patreon.button.1=打开Patreon界面
+booklet.actuallyadditions.chapter.patreon.text.2=只有<item>3像素<r>的<imp>Patreon回报<r>。
+
+booklet.actuallyadditions.chapter.changelog.name=更新日志
+booklet.actuallyadditions.chapter.changelog.text.1=你是否对<item>Actually Additions<r>曾经所有版本的<imp>变化<r>抱有兴趣？<n>如果是的话，只需要点击下面的按钮就可以看到所有<item>更新日志<r>的细节，包括<imp>添加的记录<r>，所有<imp>bug修复记录<r>，以及所有其他类型的<imp>变化<r>！
+booklet.actuallyadditions.chapter.changelog.button.1=打开更新日志界面
+
+booklet.actuallyadditions.chapter.curse.name=模组网站
+booklet.actuallyadditions.chapter.curse.text.1=你是否想要<imp>下载<r>这个模组的最新版本，查看有关这个模组的<imp>最新信息<r>，那我非常建议你点击下下这个<imp>CurseForge页面<r>的按钮。
```
Comment (SihenZhang) [inline src/main/resources/assets/actuallyadditions/lang/zh_CN.lang]: 点击下下这个<imp>CurseForge页面<r>的按钮。

两个“下”

**Comment by Ellpeck** [comment]
That was the wrong branch! Come on, people >_>

**Comment by TartaricAcid** [comment]
Waiting? Does the main branch is 1.11?

**Comment by Ellpeck** [comment]
You did these changes on the 1.10.2 branch which 1.) by the looks of it, won't be getting any updates anymore at all because there seems to be no significant bug reports coming up 2.) will never be merged into the 1.11 branch (which is the main branch) :v

**Comment by TartaricAcid** [comment]
Oh, I thought you had two branches, one is 1.10, and the main is 1.11.
XD, I'll make a correction.

**Comment by Ellpeck** [comment]
I do have two branches, and yes, one is 1.10 and the other one is 1.11. But 1.10 is the old, outdated branch ;)

**Comment by TartaricAcid** [comment]
I'm extremely sorry for my carelessness. XD

**Pull request merged**